<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vue_cartes</class>
 <widget class="QWidget" name="Vue_cartes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="vues_cartes_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1941</width>
     <height>1141</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_retour"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>1061</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Layout_cartes"/>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>561</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Layout_joueurs"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>890</y>
     <width>221</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_ok"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
